<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Chunhui's Resume</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
        <base target="_blank">
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Chunhui Gu</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded mx-auto mb-2" src="assets/img/DSC01997.JPG" alt="" /></span>
            </a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Research Interests</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Awards</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#certificates">Self taught courses & Certificates</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#support">Additional support documents</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Chunhui
                        <span class="text-primary">Gu</span>
                    </h1>
                    <div class="subheading mb-5">
                        7900 Cambridge St, Houston, Texas 77054 · (404) 952-6630 ·
                        <a href="mailto:chunhui.gu@uth.tmc.edu">chunhui.gu@uth.tmc.edu</a>
                    </div>
                    <p class="lead mb-5">After five years as a medical student, 
                        I found myself more interested in and good at quantitative sciences and programming. 
                        Still love medicine but have felt tired about cramming style education in medicine, 
                        I wish I can boost the development of medicine from another aspect with my multi-disciplinary background.
                        Considered myself as "full-stack" biomedical data scientist with 5 years of experience in biostatistics, computer science, and bioinformatics.
                        I am currently a PhD candidate in Biostatistics at UTHealth School of Public Health, and working jointly as a GRA in MD Anderson biostatistics
                        and Clinical Cancer Prevention Department.
                    </p>
                    <div class="social-icons">
                        <a class="social-icon" href="https://www.linkedin.com/in/chunhui-gu-60b524177/"><i class="fab fa-linkedin-in"></i></a>
                        <a class="social-icon" href="https://github.com/FDUguchunhui"><i class="fab fa-github"></i></a>
                        <a class="social-icon" href="https://scholar.google.com/citations?user=xB7f1ycAAAAJ&hl=en">
                            <img src="assets/img/Google_Scholar_logo.svg.png" alt="Google Scholar" width="32" height="32">
                        </a>
                        <a class="social-icon" href="#" style="opacity:0.5"><i class="fab fa-facebook-f"></i></a>
                    </div>
                </div>
            </section>

            <!-- Interests-->
            <section class="resume-section" id="interests">
                <div class="resume-section-content">
                    <h2 class="mb-5">Interests</h2>
                    <p>
                        I am keen on adapting advanced statistical and computer science techniques, as well as devising novel computational statistical methods, to address challenges in biology and medicine.
                    </p>
                </div>
            </section>

            <hr class="m-0" />
            <!-- Experience-->

            <section class="resume-section" id="experience">  
                <div class="resume-section-content">
                    <h2 class="mb-5">Experience</h2>
                    
                    <!-- Experience 1-->
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-6">the Department of Biostatistics at The University of Texas MD Anderson Cancer Center </div>
                            <div class="subheading mb-3">
                                Advisor: Prof. Irajizad Ehsan, Ph.D <br> Supervisor: Prof. Kim-Anh Do, Ph.D
                            </div>
                            <article>
                                <section>
                                    <div style="font-weight:bold"><a href=propensity.pdf>In-depth proteomic profiling of gastric cancer tumor cells uncovers previously unidentified cancer testis antigens</a></div>
                                    <ul>
                                        <li>Learned structure and properties of mass spectrometry proteomics data</li>
                                        <li>Built in-house R programming package for automating proteomics analysis pipeline including special data structure, statistical testing, and visualization modules</li>
                                        <li>Identified missing proteins using combined information in mass spectrometry protein expression and mRNA expression </li>
                                    </ul>
                                </section>
            
                                <br>
            
                                <sectio>
                                    <div style="font-weight:bold">Using graph neural networks with protein-protein-interaction for enhancing protein identification</div>
                                    <ul>
                                        <li>Reviewed status quo in using information from other sources to improve mass spectrometry-based protein detection</li>
                                        <li>Learned current state-of-art graph neural network models, such as GCNConv, GraphSAGE, and GAT</li>
                                        <li>Implemented graph neural network deep learning framework for enhancing mass spectrometry proteomics protein detection with protein-protein interaction information</li>
                                    </ul>
                                </sectio>
                                
                                <br>

                                <section>
                                    <div style="font-weight:bold">Does Cancer History Drive COVID-19 Outcome? A Large-scale Matched Cohort Analysis</div>
                                    <ul>
                                        <li>Debugged SAS code from other biostatisticians</li>
                                        <li>Translated SAS code into R code to test the compatibility of R and SAS in our routine analyses</li>
                                        <li>Developed R in-house packages to automate analysis pipeline </li>
                                        <li>Offered additional expertise from clinical medicine and biostatistics perspective in topic development and modeling</li>
                                    </ul>
                                </section>

                            </article>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">July 2022 - Current</span></div>
                    </div>

                    <!-- Experience 2-->
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-6">Texas Institute for Restorative Neurotechnologies</div>
                            <div class="subheading mb-3">Advisor: Prof. Guoqiang Zhang, Ph.D.</div>
                            <article>
                                <section>
                                    <div style="font-weight:bold"><a href=propensity.pdf>A review of propensity-score matching method in studies that exploring factors associated with COVID-severity</a></div>
                                    <ul>
                                        <li>Reviewed use of propensity-score matching method in selected papers</li>
                                        <li>Generalized commonly used propensity-score models, matching strategies, and covariates used in this topic</li>
                                        <li>Summarized shortcomings in using propensity-score matching in this topic </li>
                                    </ul>
                                </section>
            
                                <br>
            
                                <sectio>
                                    <div style="font-weight:bold">Studied how to query through MongoDB database and explore the effect of hydroxychloroquine in treating COVID-19 by propensity-score matching analysis </div>
                                    <ul>
                                        <li>Performed basic MongoDB operations, aggregation framework, improving MongoDB performance, MongoDB Python developer, and MongoDB diagnostics and debugging</li>
                                        <li>Studied theory and application of propensity-score methods in covariate selection, missing imputation, propensity-score model selection, diagnostics, and analysis</li>
                                        <li>Reviewed published papers in exploring effect of hydroxychloroquine in treating COVID-19</li>
                                        <li>Two curriculum projects were done for practice purposes:
                                            <ul>
                                                <li><b>Stochastic process I:</b> <a href=Stochastic.pdf>Comparing the length of stay of COVID-19 patients receiving and not receiving hydroxychloroquine treatment using discrete-time Markov chain</a></li>
                                                <li><b>Multivariate analysis:</b> <a href=multivariate.pdf>Estimating the effect of preventative use of hydroxychloroquine and interaction between age in four severity biomarkers after COVID infection</a></li>
                                              </ul>
                                        </li>
                                    </ul>
                                </sectio>
                                
                                <br>

                                <section>
                                    <div style="font-weight:bold">Does Cancer History Drive COVID-19 Outcome? A Large-scale Matched Cohort Analysis</div>
                                    <ul>
                                        <li>Debugged SAS code from other biostatisticians</li>
                                        <li>Translated SAS code into R code to test the compatibility of R and SAS in our routine analyses</li>
                                        <li>Developed R in-house packages to automate analysis pipeline </li>
                                        <li>Offered additional expertise from clinical medicine and biostatistics perspective in topic development and modeling</li>
                                    </ul>
                                </section>

                            </article>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2020 - May 2022</span></div>
                    </div>


                    <!-- Experience 3-->
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Assistant Bioinformatics Analyst</h3>
                            <div class="subheading mb-6">Children’s Hospital of Atlanta</div>
                            <div class="subheading mb-3">Advisor: Prof. Rabindra M Tirouvanziam, Ph.D.</div>
                            <article>
                                <ul>
                                    <li>Processed raw data from microarray and RNA-seq using R (beadarray) and Linux command line (bowtie)</li>
                                    <li>Analyzed microarray and RNA-seq data by differential expression analysis and genes enrichment analysis using R (limma and DESeq2). Summarized analysis results with chromosome ideogram, heatmap, bubble chart, and other plots by JavaScript, Python, and R</li>
                                    <li>Built well-documented in-house R packages for automated RNA-seq data analysis and plotting</li>
                                    <li>Conducted research genes associated with a good prognosis in severe influenza infection by PCA, ingenuity pathway, modular analysis, Cibersort, and etc</li>
                                </ul>
                            </article>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">May 2019 - August 2020</span></div>
                    </div>



                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Research Assistant</h3>
                            <div class="subheading mb-6">Department of Biostatistics and Bioinformatics, Emory University</div>
                            <div class="subheading mb-3">Advisor: Prof. Nelson Chen, Ph.D.</div>
                            <section>
                                <div style="font-weight:bold">Expansion of a Risk Prediction Model for Healthcare facility-onset Clostridioides difficile infection in Patients Receiving Systemic Antibiotics </div>
                                <ul>
                                    <li>Conducted ANOVA to compare mean of covariates of patients among case and control group, constructed multivariate logistic regression model with variables that significantly associated with hospital-onset CDI and evaluable at hospital admission.</li>
                                    <li>Evaluated regression model and point-based risk prediction model by receiver operating characteristic curve (ROC curve), positive predictive value, negative predictive value, sensitivity, specificity, and accuracy at various point cutoffs.</li>
                                </ul>
                            </section>
        
                            <br>
        
                            <sectio>
                                <div style="font-weight:bold">Oral Thiazide Diuretic Comparison in Acute Decompensated Heart Failure </div>
                                <ul>
                                    <li>Compared efficacy and safety of two thiazide diuretic in weight change, 24-hour-urine-output (UOP), and length of stay with Chi-square and ANCOVA in SAS.</li>
                                    <li>Used multivariate logistic regression model to analyze ICU transfer rate between two groups after adjusting for other factors.</li>
                                </ul>
                            </sectio>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">April 2018 - May 2020</span></div>
                    </div>


                    
                </div>
            </section>
            <hr class="m-0" />
            <!-- Education-->
            <section class="resume-section" id="education">
                <div class="resume-section-content">
                    <h2 class="mb-5">Education</h2>

                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">University of Texas Health Science Center at Houston</h3>
                            <div class="subheading mb-3">Doctor of Philosophy</div>
                            <div>Biostatistics</div>
                            <p>GPA: 3.916/4.0</p>
                            <p>Core courses: Generalized Linear Regression, Linear Models, Stochastic Process, Statistical Computating</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2020 - Present</span></div>
                    </div>

                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Rollins School of Public Health, Emory University</h3>
                            <div class="subheading mb-3">Master of Science</div>
                            <div>Biostatistics</div>
                            <p>GPA: 3.961/4.0</p>
                            <p>Core courses: Probability and Distribution Theory, Statistical Inference, Lienar Regression, Categorical Analysis, Survival Analysis</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">Sep 2018 - May 2020</span></div>
                    </div>

                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Fudan University</h3>
                            <div class="subheading mb-3">Bachelor of Medicine and Bachelor of Surgery</div>
                            <div>Clinical Medicine</div>
                            <p>Rank: 80/203</p>
                            <p>Core base courses: Biochemistry, Biology, Anatomy, Physiology, Imunology, Microbiology, Pharmalogy</p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">September 2013 - July 2018</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-5">Skills</h2>
                    <div class="subheading mb-3">Programming Languages & Tools</div>
                    <p>
                        <b>Statistical Software and Programming:</b> Python (mastery), R programming (mastery), Linux command line (familiar), SQL (proficiency), SAS (proficiency), Java (proficiency), C (familiar), SPSS (familiar), MongoDB (familiar)</br>
                        <b>Technique:</b> Propensity-score methods, Linear and Non-linear regression, Bayesian model, Survival analysis, Complex survey design and analysis, Clinical trials, Bootstrap, Times series, Differential expression analysis</br>
                        <b>Software:</b> Microsoft Office (Word, Excel, and PowerPoint), Photoshop, Premiere, and Lightroom</br>
                    </p>
                </div>
            </section>
            <hr class="m-0" />
            
            <hr class="m-0" />
            <!-- Awards-->
            <section class="resume-section" id="awards">
                <div class="resume-section-content">
                    <h2 class="mb-5">Awards</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Fudan University Outstanding Graduate Student Scholar 2018
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            3<sup>rd</sup> prize winning contest group in Clove Programming Competition 2016
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Fudan University Academic Excellent Scholarship 2013/2014/2015
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Excellent Volunteer of Shanghai Children’s Museum 2014
                        </li>
                    </ul>
                </div>
            </section>

            <!--publications-->
            <section class="resume-section" id="publications">
                <div class="resume-section-content">
                    <h2 class="mb-5">Publications</h2>
                    <div>
                        <p><a>Gu, C., Cai, Y., Hsiao, F.-C., Dennison, J., Vykoukal, J., Farhmann, J., Do, K.-A., Ajani, J. A., Katayama, H., Irajizad, E., & Hanash, S. (Addressing reviews). In-depth proteomic profiling of gastric cancer tumor cells uncovers previously unidentified cancer testis antigens. Cells.</a></p>
                        <p><a href=https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0267584>Kim, Y., Zhu, L., Zhu, H., Li, X., Huang, Y., Gu, C., Bush, H., Chung, C., & Zhang, G.-Q. (2022). Characterizing cancer and COVID-19 outcomes using electronic health records. PLoS One, 17(5). https://doi.org/10.1371/journal.pone.0267584</a></p>
                        <p><a href=https://www.sciencedirect.com/science/article/pii/S2666379121000550?via%3Dihub>Margaroli C, Moncada-Giraldo D, Gulick DA, Dobosh B, Giacalone VD, Forrest OA, Sun F, <b>Gu C</b>, Gaggar A, Kissick H, Wu R, Gibson G, Tirouvanziam R. Transcriptional firing represses bactericidal activity in cystic fibrosis airway neutrophils. Cell Rep Med. 2021 Apr 8;2(4):100239. doi: 10.1016/j.xcrm.2021.100239. PMID: 33948572; PMCID: PMC8080108.</a></p>
                        <p><a href=https://pubmed.ncbi.nlm.nih.gov/33954428/>Tilton, C., Elizabeth, M., Johnson, S., Chen, Z., <b>Gu, C.</b>, Robichaux, C., Metzger, N.Evaluation of a risk assessment model to predict infection with healthcare facility-onset Clostridioides difficile infection. Available at: https://pubmed.ncbi.nlm.nih.gov/33954428/</a></p>
                    </div>
                </div>
            </section>
            
            <!--for certification-->
            <section class="resume-section" id="certificates">
                <div class="resume-section-content">
                    <h2 class="mb-3">Self-taught courses & Certifications</h2>
                    <div class="subheading mb-3">Click to check corresponding certificate or study notes</div>
                    <ul class="fa-ul mb-0">
                        
                        <h5 class="mb-6">Programming Laguages</h5>
                        <li><a  href="https://coursera.org/share/40cf89a548e47ea5288336c0073e2183target" target="_blank">Programming for Everybody (Getting Started with Python) </a></li>
                        <li><a href="https://coursera.org/share/f14e0de418193ef8bcf9d2ac34044128" target="_blank">Python Data Structures|University of Michigan </a></li>
                        <li><a href="https://www.coursera.org/account/accomplishments/specialization/certificate/TSQJPTSGW78J" target="_blank">Introduction to Scripting in Python|Rice University </a>
                            <ul>
                                <li>Python Programming Essentials</li>
                                <li>Python Data Representations</li>
                                <li>Python Data Analysis</li>
                                <li>Python Data Visualization</li>
                            </ul>
                        </li>
                        <br>
                        <h5 class="mb-6">Tools</h5>
                        <li><a href="https://coursera.org/share/888ddcfd29b0c1f3bc44b65c5f89f0f4" target="_blank">Version Control with Git|Atlassian</a></li>
                        <li><a href="https://coursera.org/share/dc5d11cbcfd7f7e0c5b625065d17eafc" target="_blank">Introduction to Genomic Technologies|Johns Hopkins University</a></li>
                        <li><a href="https://coursera.org/share/2d76b2234d81fd21583c6b323ddee31a" target="_blank">Python for Genomic Data Science|Johns Hopkins University</a></li>
                        <br>
                        <h5 class="mb-6">Mathematics & Statistics</h5>
                        <li><a href="https://coursera.org/share/2585021dd2846645c0829f752e0a4768" target="_blank">Bayesian Statistics: From Concept to Data Analysis|University of California, Santa Cruz</a></li>
                        <li><a href="https://coursera.org/share/e92774f1995877dfc3336a9d0eb55952" target="_blank">Bayesian Statistics: Techniques and Models|University of California, Santa Cruz</a></li>
                        <li><a href="https://coursera.org/share/dfa6f5fb405959e7cb04b55625bfb835" target="_blank">Practical Time Series Analysis|The State University of New York</a></li>
                        <li>18.06 Linear Algebra (Spring 2010) | Gilbert Strang, Massachusetts Insistute of Technology</li>
                        <li>6.042J/18.062J Mathematics for Computer Science (Fall 2010) <span style="color:red">in progress</span>| Massachusetts Insistute of Technology</li>
                        <br>
                        <h5 class="mb-6">Computer Science</h5>
                        <li><a target="_blank">CS 7643 Deep Learning|Georgia Technology Institute</a></li>
                        <li><a href="https://credentials.edx.org/credentials/e44dfb1cb7c0426f9d25490475560bf3/" target="_blank">Data Structures and Algorithms Professional Certificate|Georgia Technology Institute</a></li>
                        <li><a href="https://certificates.cs50.io/a8d14f65-713b-4c23-8fa8-f1b932136d50.pdf?size=lette" target="_blank">CS50 Introduction to computer science|Harvard University</a></li>
                        <li><a href="https://coursera.org/share/3acde7a78d10db305eef35b5d7d1e79d" target="_blank">Data Structure|University of California, San Diego</a></li>
                        <li><a href="https://coursera.org/share/653e2bd424b5f8cdda8ee25270f6a7f3" target="_blank">Managing Big Data with MySQL|Duke University</a></li>
                        <li><a href="https://coursera.org/share/af232e89033facc90634811dab21234e" target="_blank">Machine Learning|Stanford University</a></li>
                        <br>
                        <h5 class="mb-6">Computer Science</h5>
                        <li><a href="https://university.mongodb.com/course_completion/f4e02b56-bb7d-44e6-a0b2-7061453c3e7d" target="_blank">M001: MongoDB Basics</a></li>
                        <li>M121 Aggregation framework</li>
                        <li><a href="https://university.mongodb.com/course_completion/804e1040-909b-410b-84b7-7f90f19ff894?utm_source=copy&utm_medium=social&utm_campaign=university_social_sharing" target="_blank">M201 MongoDB performance</a></li>
                        <li><a href="https://university.mongodb.com/course_completion/eeb04f96-e457-4342-b160-a1be0484a88c" target="_blank">M220P Python developer</a></li>

                    </ul>
                </div>
            </section>

            

             <!--Additional support documents-->
             <section class="resume-section" id="support">
                <div class="resume-section-content">
                    <h2 class="mb-5">Additional support documents</h2>
                    <div>
                        <ul>
                            <li><a href="transcripts.pdf" >View Master Transcripts</a></li>
                            <li><a href="transcripts_undergraduate.pdf" >View Bachelor Transcript</a></li>
                            <li><a href="thesis_chunhui.pdf" >View My Master Thesis</a></li>
                            <li><a href="UTUNOFTRNSC1.pdf">View My Current PhD transcript</a></li>
                        </ul>
                    </div>
                </div>
            </section>

        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"></script>
        <!-- Third party plugin JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
